CHANGELOG.md

```diff
@@ -1,37 +1,113 @@
 # Changelog
 
-## 6.0.0-rc.1
+## 6.0.0
 
-**Note:** This version is a pre-release and is not recommended for production usage.
+Teleport 6.0 is a major release with new features, functionality, and bug fixes.
 
-This release of Teleport contains a number of improvements and bug fixes.
+The major features are [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open source role-based access control (RBAC), and exposing an easy to use API.
```

Comment:
`We have implemented [Database Access](https://goteleport.com/teleport/docs/preview/database-access/), open sourced role-based access control (RBAC), and added official API and a Go client library.